lib/perl/Genome/Model/ClinSeq/Util.pm

```diff
@@ -1446,5 +1446,21 @@ sub _is_copycat_somvar {
     }
 }
 
+sub get_best_somvar_build {
+    my $self = shift;
+    my $clinseq_build = shift;
+    my $somvar_build =
+        $clinseq_build->model->wgs_model->last_succeeded_build;
```

Comment:
A similar caution against getting the build via `last_succeeded_build` here.  `$clinseq_build->wgs_build` would be more direct.